components/calendar/Cargo.toml

```diff
@@ -25,7 +25,8 @@ include = [
 
 [features]
 std = []
-provider_serde = ["serde", "litemap/serde_serialize", "zerovec/serde", "tinystr/serde"]
+serialize = ["serde", "litemap/serde", "zerovec/serde", "tinystr/serde"]
```

Comment:
Question: You said feature "serialize" is going to be renamed to "serde" as soon as we're able to do that, correct?

Reply:
Correct, in a couple weeks. I'll file a followup and assign myself once this merges.

Reply:
https://github.com/unicode-org/icu4x/issues/1708